static/js/mediastream/peerconference.js

```diff
@@ -249,7 +261,11 @@ define(['jquery', 'underscore', 'mediastream/peercall'], function($, _, PeerCall
 
 	PeerConference.prototype.peerIds = function() {
 
-		return _.keys(this.calls);
+		var result = _.keys(this.calls);
+		if (this.currentcall && this.currentcall.id && result.indexOf(this.currentcall.id) === -1) {
```

Comment:
Add a comment why it is required to check for currentcall.id in addition to currentcall.